cpp/include/cudf_test/cxxopts.hpp

```diff
@@ -17,6 +17,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
+#pragma once
+
```

Comment:
This file is vendored, and it uses a different header guard (`#ifndef CXXOPS_HPP_INCLUDED`, `#define CXXOPTS_HPP_INCLUDED`). Let's revert this change.
```suggestion
```